package.json

```diff
@@ -49,7 +49,6 @@
     "@typescript-eslint/eslint-plugin": "^4.31.2",
     "@typescript-eslint/parser": "^4.0.1",
     "eslint": "^7.32.0",
-    "eslint-config-airbnb-base": "^14.2.1",
```

Comment:
we don't really need this plus there is a conflict in dependency versions because of this